### PR TITLE
Add automated check for OBO Library PURL configuration

### PR DESCRIPTION
**Pull request by cthoyt**
As mentioned in https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1958#issuecomment-1249287870, all new ontologies should have a working configuration on the OBO Library PURL service at https://github.com/OBOFoundry/purl.obolibrary.org before merging any new ontology metadata files.

This PR adds this check, which formats the prefix for the ontology into the appropriate URL and checks if it can be retrieved with a HTTP 200.

## Blocked by

- [x] @sabrinatoro you should create an appropriate configuration, noted in https://github.com/OBOFoundry/purl.obolibrary.org/issues/869

### CODE REVIEW

**Comment by matentzn** [comment]
I think this makes sense, but for some reason I remember that there is a dependency the other way around (purl-->metadata). @jamesaoverton can you confirm/unconfirm my memory? I somehow remember you telling me 2 years ago that we first merge metadata than purl configs, but I don't remember why or even if you said that..

**Comment by jamesaoverton** [comment]
Since the discussion takes place here, I have been waiting for the metadata file to be merged here before I merged the PURL config. As long as the process is clear, either way works.

**Comment by cthoyt** [comment]
An alternative is to add a required metadata field that's the pull request number for adding the metadata configuration to the other repo. Up to you

**Review by jamesaoverton — APPROVED** [review]
I'm happy with this. From now on the PURL PR for a new ontology will have to be merged before the registry entry PR, which makes sense.

**Comment by matentzn** [comment]
@cthoyt PR is currently failing because of the python requirements that are missing. 

Also in the error log I am noticing something entirely unrelated:

```
INFO	NOMEN funded_by: 'url' is a required property
INFO	UBERON funded_by: 'url' is a required property
INFO	ECO funded_by: 'url' is a required property
```

Shouldnt this fail with all your work you put into standardising funding information?

**Comment by matentzn** [comment]
Thanks a ton :)

**Pull request merged**
